darts/models/ensemble_model.py

```diff
@@ -45,14 +45,14 @@ def fit(self, series: TimeSeries) -> None:
     def predict(self, n: int) -> TimeSeries:
         super().predict(n)
 
-        predictions = []
-        for model in self.models:
-            predictions.append(model.predict(n))
+        predictions = self.models[0].predict(n)
+        for model in self.models[1:]:
```

Comment:
only do this if there are >1 models otherwise it might fail.

Reply:
Done